test/e2e/storage/csi_volumes.go

```diff
@@ -83,6 +86,11 @@ var _ = utils.SIGDescribe("CSI Volumes", func() {
 			WaitForCompletion: true,
 		}
 		csiDriverRegistrarClusterRole(config)
+		createCSICRDs(crdclient)
```

Comment:
This prevents running tests in parallel, was that already the case?

Reply:
Nope. I don't see a way out of a CRD installation race condition so I'll go ahead and mark it serial, and leave a TODO to remove it when CRD is installed automatically

Reply:
Not AFAIK. There's `SynchronizedBeforeSuite()`:
https://github.com/kubernetes/kubernetes/blob/e7eb26919b8ebd2b1c86e58cdfc4f7db1edf4c2e/vendor/github.com/onsi/ginkgo/ginkgo_dsl.go#L496

but it can only be set once and it's already set here:
https://github.com/kubernetes/kubernetes/blob/e7eb26919b8ebd2b1c86e58cdfc4f7db1edf4c2e/test/e2e/e2e.go#L137

Given that we aim to have a generic CRD auto-install solution for different CRDs, I don't think it's worth the effort to devise a CRD installation mechanism in this function that reference specific tests, only to have it removed in the next release.

@kubernetes/sig-testing are there any other suggestions for installing CRDs once before any tests in the suite are run in parallel? Otherwise making the tests Serial might be our best bet

Reply:
the problem is deletion right? Could we just not delete the CRD's from the cluster or would that cause issues...?